event/handler.go

```diff
@@ -176,7 +178,10 @@ func (handler *ExportHandler) isVersionPublished(ctx context.Context, event *Fil
 }
 
 func (handler *ExportHandler) filterJob(event *FilterSubmitted, isPublished bool) (*CSVExported, error) {
-	log.Info("handling filter job", log.Data{"filter_id": event.FilterID})
+
+	ctx := context.Background()
```

Comment:
We shouldn't be generating a context every time we need to add one to the logs, this should be passed into method